### PR TITLE
Carousel 2

### PR DESCRIPTION
**Pull request by b3an5**
**fixed merge conflicts**

I made it so that when you press the left or right arrows you can move to the team in said direction. it has to switch back to team before we move so there could be a potential addition down the road to make it so we can just cycle through the cities alone.

### CODE REVIEW

**Comment by b3an5** [comment]
http://designerstoolbox.com/designresources/html/
also found this website that yall might be interested in

File: src/Container.js [hunk 1]
```diff
@@ -30,6 +30,30 @@ export default class Container extends Component {
     this.getIndex(index)
   }
 
+  teamCarouselRight = () => {
+    if (this.state.clickedIndex !== 31) {
```
Comment (ericfitzsimons451) [inline src/Container.js]: Glad that we have that clicked index!  Smooth way to move through the data!

File: src/Container.js [hunk 2]
```diff
@@ -55,6 +79,8 @@ export default class Container extends Component {
                 cities={this.props.cities}
                 toggleCardView={this.toggleCardView}
                 clickedIndex={this.state.clickedIndex}
+                teamCarouselLeft={this.teamCarouselLeft}
```
Comment (ericfitzsimons451) [inline src/Container.js]: Let's make sure to add some verbs to our naming conventions

File: src/Card.js [hunk 3]
```diff
@@ -40,6 +50,8 @@ export default class Card extends Component {
           <h2 className='team-info' >{selectedTeam.city}, {selectedTeam.state}</h2>
           <h2 className='team-info' >{selectedTeam.division}</h2>
           <button onClick={this.findMatchingCity} className="card-btn">Show City Info</button>
+          <h1 className='left-carot carot' onClick={this.props.teamCarouselLeft}>&laquo;</h1>
+          <h1 className='right-carot carot' onClick={this.props.teamCarouselRight}>&raquo;</h1>
```
Comment (geet084) [inline src/Card.js]: that's slick, html ftw!

**Pull request merged**